docs/GeneratorOptions.md

```diff
@@ -0,0 +1,70 @@
+# vets-website Yeoman generator
+
+We have a Yeoman generator for starting up new React applications in vets-website. This page will guide you through setting up the generator and explaining the questions that are asked as part of the setup process.
+
+## Getting Started
+
+To install the generator, you'll need to do the following:
+
+```bash
+npm install -g yo
+npm install -g @department-of-veterans-affairs/generator-vets-website
+```
+
+Once you have that set up, navigate to the root of the vets-website repository and run:
+
+```bash
+yo @department-of-veterans-affairs/vets-website
+```
+
+## Generating an application
+
+After running the above command, the generator will start up and ask you a series of questions:
+
+### What's the name of your application?
+
+This value will be used as the page title. If you're creating a form, it will also be used as the header for the introduction page and as a header on the confirmation page.
+
+### What folder in src/js should your app live in?
+
+Most of our React applications have their own folder in src/js, so normally you want to pick a new folder. However, if you're building an application that's related to other applications, there may be a folder that your app should go in. An example would be if you were creating a new education form, you would probably create your app in edu-benefits/newform.
+
+### What should be the name of your app's entry bundle?
+
+This is the name of the bundle that Webpack builds for your application. They're normally lower case with dashes separating words. This is primarily used to link your bundle to the content page that is created as a base for your application. It doesn't show up in your code anywhere. It's also separate from the entry file.
+
+### What's the root url for this app?
+
+This is the url your application will live at. In your React apps you will likely have multiple pages and the urls for those pages will have this as the base. This value also gets translated into the page for the content page, which lives in `content/pages`.
+
+### Is this a form app?
+
+If this is a form, there are some more questions after this step. If not, you're done!
+
+### What's your form number?
+
+This is the form number for the paper form you're converting. Normally it's something like 22-1990 or 21-22. This is used in a couple places in the UI and also as a key for the save in progress functionality of our forms.
```

Comment:
One suggestion here would be to clarify where you might find the form number on the paper form, or even include a screenshot, like so: 
![screen shot 2018-04-05 at 4 40 27 pm](https://user-images.githubusercontent.com/19768529/38390724-28636fe2-38f0-11e8-8004-2db532cfa3bc.png)

Reply:
I see now that this has been handled further below!